src/khoj/interface/web/config.html

```diff
@@ -3,6 +3,25 @@
 
 <div class="page">
     <div id="content" class="section">
+        <h2 class="section-title">Profile</h2>
+        <div class="section-cards">
+            <div class="card">
+                <div class="card-title-row">
+                    <img class="card-icon" src="/static/assets/icons/user-silhouette.svg" alt="Chat">
+                    <h3 class="card-title">
+                        Name
+                    </h3>
+                </div>
+                <div class="card-description-row">
+                    <input type="text" id="profile_given_name" class="form-control" placeholder="Michael Jordan" value="{{ given_name }}">
```

Comment:
Should placeholder be set to their current/default username (=username associated with gmail address). It can also be an instruction like "Enter your name"?
```suggestion
                    <input type="text" id="profile_given_name" class="form-control" placeholder="Michael Jordan" value="{{ given_name }}">
```

Reply:
Given name is a computed value which would be the default user name if an explicit one isn't provided. See method named `get_user_name`.